qdrant/v1.2.x/collections.md

```diff
@@ -59,7 +59,14 @@ In addition to the required options, you can also specify custom values for the
 
 Default parameters for the optional collection parameters are defined in [configuration file](https://github.com/qdrant/qdrant/blob/master/config/config.yaml).
 
-See [schema definitions](https://qdrant.github.io/qdrant/redoc/index.html#operation/create_collection) and a [configuration file](https://github.com/qdrant/qdrant/blob/master/config/config.yaml) for more information about collection parameters.
+See [schema definitions](https://qdrant.github.io/qdrant/redoc/index.html#operation/create_collection) and a [configuration file](https://github.com/qdrant/qdrant/blob/master/config/config.yaml) for more information about collection and vector parameters.
+
+*Available as of v1.2.0*
+
+The `on_disk` parameter can be set in the vector configuration. If true, all
+vectors will be stored on disk at all times. This will enable the use of
```

Comment:
I find our usage of `store` in the docs more and more confusing.
Are there times where the vectors are not stored on disk?

My understanding is that the vectors are always persisted on disk but they are sometimes loaded in memory for performance reason.

WDYT?

Reply:
> Are there times where the vectors are not stored on disk?

That is a very good point. You're right.

The thing is, that with mmap (`on_disk = true`), vectors are also in memory (most of the time).

I'm not sure how to phrase this better. I don't think the following makes it clear:

> Normally vectors are primarily in RAM. With appendable mmap vectors are primarily on disk.

Reply:
Agreed in private chat that this is better. Marking as resolved. :+1: